BUILD

```diff
@@ -36,17 +36,36 @@ this file by hand, before proceeding with the build. Note that the the generated
 a symbolic link to a system-specific default config file. It is not necessary to edit the file; you
 can override variables on the "make" command line if you prefer.
 
-An alternative to "make mconfig" is to use the provided "configure" script. It will try to
-generate a suitable "mconfig" file, based on sensible defaults and options provided on the command
-line when the script is run.
+An alternative to "make mconfig" is to use the provided "configure" script. See "configure script",
+below.
+
+You can also create and edit the "mconfig" file completely by hand (or start by copying one for a
+particular OS from the "configs" directory) to choose appropriate values for the configuration
+variables defined within.
+
+
+"configure" script
+=-=-=-=-=-=-=-=-=-
+
+"configure" is a script to generate a suitable "mconfig" file, based on sensible defaults and
+options provided on the command line when the script is run. It also has checks to enable/disable
+certain features based on the availability of their requirements (such as presence of certain
+header files).
 
 For more information on available options from the configure script, run:
 
     ./configure --help
 
-You can also create and edit the "mconfig" file completely by hand (or start by copying one for a
-particular OS from the "configs" directory) to choose appropriate values for the configuration
-variables defined within.
+One of the checks performed by "configure" is the "-fno-rtti" check, which involves compiling
+a test program and running it to ensure that the "-fno-rtti" compiler option works correctly.
+However, when building Dinit for another platform (i.e. cross-compiling), because of the inability
+to run the test program on the target machine, this check cannot be performed. "configure" disables
+"-fno-rtti" for cross builds and prints out this warning:
+
+    Cross-compilation detected. Disabling -fno-rtti to make sure exceptions will always work.
+
+See "Recommended compiler options", below, for more information about the "-fno-rtti" compiler
+option and things to be considered when using it.
```

Comment:
This is too late, you've already talked about the `-fno-rtti` option without explaining it, you need a parenthetical there. I pointed out one specific case of this earlier, but you missed this one.

Also there should be a cross-reference to the new "Special note" section (as an explanation of why this test program is needed).